### PR TITLE
inital additions to docs/intro_tutorial.md

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

**Comment by rabernat** [comment]
Let us know when you're ready for feedback here.

**Comment by norlandrhagen** [comment]
Feedback would be great! Especially the end of the document.

File: docs/intro_tutorial.md [hunk 1]
```diff
@@ -13,10 +13,212 @@ To create your ARCO dataset, you'll need to create a new recipe.
...
+![Architecture: High Level View](https://github.com/pangeo-forge/flow-charts/blob/main/renders/architecture.png?raw=true)
 
-TODO
 
-### Maintaining a Recipe
+See [maintaining](##maintaining) for more on what happens next.
 
-TODO
+
+# Initial Directory Structure Setup / Forking
```
Comment (rabernat) [inline docs/intro_tutorial.md:22]:
```suggestion
## Initial Directory Structure Setup / Forking
```

**Comment by rabernat** [comment]
I'm going to merge this for now so we can get something up. Let's keep iterating on this page via future PRs.

**Pull request merged**